public/manifest.json

```
@@ -1,19 +1,22 @@
 {
-    "name": "",
-    "short_name": "",
+    "name": "rezervo",
+    "short_name": "rezervo",
     "icons": [
         {
             "src": "/android-chrome-192x192.png",
             "sizes": "192x192",
-            "type": "image/png"
+            "type": "image/png",
+            "purpose": "any maskable"
         },
         {
             "src": "/android-chrome-512x512.png",
             "sizes": "512x512",
             "type": "image/png"
         }
     ],
-    "theme_color": "#ffffff",
+    "theme_color": "#4caf50",
     "background_color": "#ffffff",
-    "display": "standalone"
+    "start_url": "/",
+    "display": "standalone",
+    "orientation": "portrait"
```

Need to remove `orientation` for landscape support (at least on Android, not tested on iOS). Seems [experimental](https://developer.mozilla.org/en-US/docs/Web/Manifest/orientation) anyway.
```suggestion
    "display": "standalone"
```

---

Removed it now